cli/pcluster/config/mappings.py

```diff
@@ -135,7 +136,7 @@
     "snapshot_id": r"^snap-[0-9a-z]{8}$|^snap-[0-9a-z]{17}$",
     "subnet_id": r"^subnet-[0-9a-z]{8}$|^subnet-[0-9a-z]{17}$",
     "volume_id": r"^vol-[0-9a-z]{8}$|^vol-[0-9a-z]{17}$",
-    "volume_types": ["standard", "io1", "gp2", "st1", "sc1"],
+    "volume_types": ["standard", "io1", "io2", "gp2", "st1", "sc1"],
```

Comment:
let's remove io2 from this list for now. the io2 logic can remain in the validators, and the test cases can be commented out with a TODO comment explaining that they should be enabled when io2 support is added

Reply:
Done